artifacts/definitions/Windows/Utils/FetchBinary.yaml

```diff
@@ -17,8 +17,7 @@ parameters:
 sources:
   - queries:
       - |
-        LET download = SELECT FullPath as TempPath,
-            expand(path=binPath) + "\\" + basename(path=binaryURL) as FullPath,
+        LET download = expand(path=binPath) + "\\" + basename(path=binaryURL) as FullPath,
```

Comment:
I dont think this is valid VQL - LET must be followed by select

Reply:
it worked for me testing via the gui in 0.32 release.
![image](https://user-images.githubusercontent.com/13081800/63222181-b0ad2000-c193-11e9-8cc0-686047a7739b.png)